.github/workflows/test_dbt_models.yaml

```diff
@@ -53,9 +53,7 @@ jobs:
       - name: Extract test results
         run: |
           python3 ../.github/scripts/transform_dbt_test_results.py \
-            ./target/run_results.json \
-            ./target/manifest.json \
```

Comment:
These are the default values; previously we had to provide them anyway in order to set the output directory since all the arguments were positional, but now that all the arguments are set via options, we don't need them anymore.